paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletPaymentMethodMenu.kt

```diff
@@ -14,14 +15,19 @@ internal fun WalletPaymentMethodMenu(
     paymentDetails: ConsumerPaymentDetails.PaymentDetails,
     onSetDefaultClick: () -> Unit,
     onRemoveClick: () -> Unit,
-    onCancelClick: () -> Unit
+    onCancelClick: () -> Unit,
+    onUpdateClick: () -> Unit,
 ) {
     val items = remember(paymentDetails) {
         buildList {
             if (!paymentDetails.isDefault) {
                 add(WalletPaymentMethodMenuItem.SetAsDefault)
             }
 
+            if (FeatureFlags.enableCardEditInLinkNative.isEnabled && paymentDetails is ConsumerPaymentDetails.Card) {
```

Comment:
There's no extra conditions to show the update screen on iOS rather than the PM being a card. Replicating that.